src/test/scala/chiseltest/simulator/PeekPokeCompliance.scala

```diff
@@ -70,4 +70,27 @@ abstract class PeekPokeCompliance(sim: Simulator, tag: Tag = DefaultTag) extends
     dut.finish()
   }
 
+  it should "propagate values of variable widths through the circuit" taggedAs(tag) in {
+    val Widths = Seq(1, 2, 3, 5, 7, 8, 9, 15, 16, 17, 31, 32, 33, 63, 64, 65, 127, 128, 129,
```

Comment:
Just a note that treadle's test suite has functions that generates numbers like this in src/test/scala/treadle/TestUtils.scala `BigIntTestValuesGenerator` and others
I really ought to expose those somehow